src/Sentry/ISpanTracer.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace Sentry
+{
+    /// <summary>
+    /// Span tracer.
+    /// </summary>
+    public interface ISpanTracer : ISpan
```

Comment:
ISpanTracer and ITransactionTracer add a suffix to the API, what was the inspiration for it?

Reply:
I think the main difference in this version is that

ISpan - immutable; Protocol.Span implements ISpan
ISpanTracer - ISpan + mutations (including StartChild); SpanTracer implements ISpanTracer; potentially NoOpSpanTracer would also implement it.

and then essentially the same with ITransaction/ITransactionTracer.